draft-irtf-cfrg-vdaf.md

```diff
@@ -217,48 +227,43 @@ fraction of people have experienced a given disease.  Thus systems that provide
 aggregate statistics while protecting individual measurements can deliver the
 value of the measurements while protecting users' privacy.
 
-Most prior approaches to this problem fall under the rubric of "differential
-privacy (DP)" {{Dwo06}}. Roughly speaking, a data aggregation system that is
-differentially private ensures that the degree to which any individual
-measurement influences the value of the aggregate result can be precisely
-controlled. For example, in systems like RAPPOR {{EPK14}}, each user samples
-noise from a well-known distribution and adds it to their measurement before
-submitting to the aggregation server. The aggregation server then adds up the
-noisy measurements, and because it knows the distribution from whence the noise
-was sampled, it can estimate the true sum with reasonable precision.
-
-Differentially private systems like RAPPOR are easy to deploy and provide a
-useful guarantee. On its own, however, DP falls short of the strongest privacy
-property one could hope for. Specifically, depending on the "amount" of noise a
-client adds to its measurement, it may be possible for a curious aggregator to
-make a reasonable guess of the measurement's true value. Indeed, the more noise
-the clients add, the less reliable will be the server's estimate of the output.
-Thus systems employing DP techniques alone must strike a delicate balance
-between privacy and utility.
+This problem is often formulated in terms of "differential privacy (DP)"
+{{Dwo06}}. Roughly speaking, a data aggregation system that is differentially
+private ensures that the degree to which any individual measurement influences
+the value of the aggregate result can be precisely controlled. For example, in
+systems like RAPPOR {{EPK14}}, each user samples noise from a well-known
+distribution and adds it to their measurement before submitting to the
+aggregation server. The aggregation server then adds up the noisy measurements,
+and because it knows the distribution from whence the noise was sampled, it can
+estimate the true sum with reasonable accuracy.
+
+However even when the measurements are noised, collecting them in the clear
+still reveals a significant amount of information to the collector.
+Specifically, depending on the "amount" of noise a client adds to its
+measurement, it may be possible for a curious collector to make a reasonable
+guess of the measurement's true value. Indeed, the more noise the clients add,
```

Comment:
Instead of "Specifically, ... Indeed...", I might use "On the one hand... On the other hand..."